### PR TITLE
(maint) Update master branch to 3.0.0.dev.

### PR DESCRIPTION
**Pull request by andersonmills**

### CODE REVIEW

File: lib/r10k/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module R10K
-  VERSION = '2.6.0.dev'
+  VERSION = '3.0.0.dev'
```
Comment (ajroetker) [inline lib/r10k/version.rb:2]: Does this need to be updated anywhere else in CI/PE?

Comment (andersonmills) [reply lib/r10k/version.rb:2]: Not in the previous setup. No idea what you've done recently.

Comment (andersonmills) [reply lib/r10k/version.rb:2]: I take that back.

pkg_version in rubygem-r10k.rb in pe-r10k-vanagon.

**Comment by binford2k** [comment]
@justinstoller is this PR even still relevant?

**Comment by justinstoller** [comment]
no

**Pull request closed without merging**